aea/manager/project.py

```diff
@@ -209,7 +218,7 @@ def _create_private_key(
         filepath = os.path.join(self._data_dir, file_name)
         if os.path.exists(filepath) and not replace:
             return filepath
-        create_private_key(ledger, filepath)
+        create_private_key(ledger, filepath, password=self._password)
```

Comment:
Note here, password confirmation step is too onerous. We would need to assume the supplied password is sufficient.